docs/explanation/measures.md

```diff
@@ -0,0 +1,299 @@
+# Using the measures framework
+
+## Introduction
+
+The measures framework is used to calculate **quotients** (i.e. a numerator divided by a denominator) and to see how these **vary over time** and when broken down by different **groupings**.
+
+The numerators, denominators and groups are all defined using ehrQL queries, just like you would use with a dataset definition. But the way those definitions are used is a bit different.
```

Comment:
I can see that you've used "you" and "we" deliberately throughout this document. However, I think it's friendlier to maintain the illusion that we will be writing my code together.

```suggestion
The numerators, denominators and groups are all defined using ehrQL queries, just like we would use with a dataset definition. But the way those definitions are used is a bit different.
```

Reply:
I think I just used whatever felt most natural at the time I happened to be writing so I'm not sure it's particularly consistent. Thanks for picking up on these.